airbyte-workers/src/main/java/io/airbyte/workers/DefaultCheckConnectionWorker.java

```diff
@@ -66,11 +66,15 @@ public DefaultCheckConnectionWorker(final IntegrationLauncher integrationLaunche
   @Override
   public StandardCheckConnectionOutput run(StandardCheckConnectionInput input, Path jobRoot) throws WorkerException {
 
-    final JsonNode configDotJson = input.getConnectionConfiguration();
-    IOs.writeFile(jobRoot, WorkerConstants.SOURCE_CONFIG_JSON_FILENAME, Jsons.serialize(configDotJson));
+    // Remove the standard check connection input.
+    // Remove write here.
+
+     final JsonNode configDotJson = input.getConnectionConfiguration();
+     IOs.writeFile(jobRoot, WorkerConstants.SOURCE_CONFIG_JSON_FILENAME,
```

Comment:
I was testing with these lines commented out.  had to add them back since we require them the first time we create a source.